integrated_channels/utils.py

```diff
@@ -501,3 +501,13 @@ def get_enterprise_client_by_channel_code(channel_code):
         'canvas': CanvasAPIClient,
     }
     return _enterprise_client_model_by_channel_code[channel_code]
+
+
+def dummy_reverse(_apps, _schema_editor):
+    """
+    Reverse a data migration but do nothing.
+
+    :param _apps:
+    :param _schema_editor:
+    :return:
```

Comment:
[suggestion] A better way to achieve this would be to use `RunPython.noop` as a reverse function. https://stackoverflow.com/questions/66479161/what-does-runpython-noop-do

Reply:
@sameenfatima78 suggested method can be better.